### PR TITLE
Feature/5667 fix license implementation

### PR DESCRIPTION
**Pull request by HavierD**

### CODE REVIEW

File: indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package au.org.aodn.esindexer.model;
+
+import lombok.Getter;
+
+@Getter
+public enum MediaType {
+    TEXT_HTML("text/html"),
+    TEXT_XML("text/xml"),
+    IMAGE_JPEG("image/jpeg"),
```
Comment (vietnguyengit) [inline indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java:9]: where is this one being used?

File: indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package au.org.aodn.esindexer.model;
+
+import lombok.Getter;
+
+@Getter
+public enum MediaType {
+    TEXT_HTML("text/html"),
+    TEXT_XML("text/xml"),
```
Comment (vietnguyengit) [inline indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java:8]: where is this one being used?

File: indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    IMAGE_JPEG("image/jpeg"),
+    IMAGE_PNG("image/png"),
+    APPLICATION_JSON("application/json"),
```
Comment (vietnguyengit) [inline indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java:11]: where is this one being used?

Comment (vietnguyengit) [reply indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java:11]: APPLICATION_JSON

File: indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java (hunk 1 above)
Comment (vietnguyengit) [reply indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java:9]: IMAGE_JPEG

File: indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java (hunk 2 above)
Comment (vietnguyengit) [reply indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java:8]: TEXT_XML

File: indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java (hunk 1 above)
Comment (HavierD) [reply indexer/src/main/java/au/org/aodn/esindexer/model/MediaType.java:9]: They are from good examples of stac collection. For now, not in use. Maybe they should be added later when they start using?

**Review by vietnguyengit — APPROVED** [review]

**Pull request merged**